include/z64player.h

```diff
@@ -187,6 +195,70 @@ typedef enum {
     /* 34 */ PLAYER_MWA_MAX
 } PlayerMeleeWeaponAnimation;
 
+// TODO: figure out MAX
+typedef enum PlayerAnimType {
+    /* 0 */ PLAYER_ANIMTYPE_0,
+    /* 1 */ PLAYER_ANIMTYPE_1,
+    /* 2 */ PLAYER_ANIMTYPE_2,
+    /* 3 */ PLAYER_ANIMTYPE_3,
+    /* 4 */ PLAYER_ANIMTYPE_4,
+    /* 5 */ PLAYER_ANIMTYPE_5,
+    // /* 6 */ PLAYER_ANIMTYPE_MAX
+} PlayerAnimType;
+
+typedef enum PlayerModelType {
+    // left hand
+    /*  0 */ PLAYER_MODELTYPE_LH_OPEN,
+    /*  1 */ PLAYER_MODELTYPE_LH_CLOSED,
+    /*  2 */ PLAYER_MODELTYPE_LH_ONE_HAND_SWORD,
+    /*  3 */ PLAYER_MODELTYPE_LH_TWO_HAND_SWORD,
+    /*  4 */ PLAYER_MODELTYPE_LH_4,
+    /*  5 */ PLAYER_MODELTYPE_LH_BOTTLE,
+    // right hand
+    /*  6 */ PLAYER_MODELTYPE_RH_OPEN,
+    /*  7 */ PLAYER_MODELTYPE_RH_CLOSED,
+    /*  8 */ PLAYER_MODELTYPE_RH_SHIELD,
+    /*  9 */ PLAYER_MODELTYPE_RH_BOW,
+    /* 10 */ PLAYER_MODELTYPE_RH_INSTRUMENT,
+    /* 11 */ PLAYER_MODELTYPE_RH_HOOKSHOT,
+    // sheath
+    /* 12 */ PLAYER_MODELTYPE_SHEATH_12,
+    /* 13 */ PLAYER_MODELTYPE_SHEATH_13,
+    /* 14 */ PLAYER_MODELTYPE_SHEATH_14,
+    /* 15 */ PLAYER_MODELTYPE_SHEATH_15,
+    // waist
+    /* 16 */ PLAYER_MODELTYPE_WAIST,
+    /* 17 */ PLAYER_MODELTYPE_17, // NULL?
+    /* 18 */ PLAYER_MODELTYPE_MAX
+} PlayerModelType;
+
+typedef struct PlayerModelIndices {
+    /* 0x0 */ u8 modelAnimType; // PlayerAnimType enum
+    /* 0x1 */ u8 leftHandType; // PlayerModelType enum
+    /* 0x2 */ u8 rightHandType; // PlayerModelType enum
+    /* 0x3 */ u8 sheathType; // PlayerModelType enum
+    /* 0x4 */ u8 waistType; // PlayerModelType enum
+} PlayerModelIndices; // size = 0x5
```

Comment:
May want to revisit this later, but fine for now.

Reply:
Revisit in what sense?

Reply:
The struct name doesn't tie particularly well to the element names.